packages/serum/src/market.ts

```diff
@@ -1720,7 +1720,22 @@ export class OpenOrders {
     marketAddress: PublicKey,
     ownerAddress: PublicKey,
     programId: PublicKey,
-  ) {
+  ): Promise<OpenOrders[]> {
+    // Try loading seed based accounts
+    const seed = marketAddress.toBase58().slice(0, 32);
+    const ooAccountPubkey = await PublicKey.createWithSeed(
```

Comment:
might be nice to abstract this out since you do it in other function as well. basically a function like `getDerivedOOAcountPubkey(ownerAddress,marketAddress)` so you dont need duplicate seed deriving logic

Reply:
yes 100% !